src/compiler/transformers/declarations.ts

```diff
@@ -1816,7 +1817,7 @@ export function transformDeclarations(context: TransformationContext) {
                 return cleanup(factory.updateEnumDeclaration(input, factory.createNodeArray(ensureModifiers(input)), input.name, factory.createNodeArray(mapDefined(input.members, m => {
                     if (shouldStripInternal(m)) return;
                     if (isolatedDeclarations) {
-                        if (m.initializer && !isLiteralExpression(m.initializer)) {
+                        if (m.initializer && (!isLiteralExpression(m.initializer) && !(isPrefixUnaryExpression(m.initializer) && isLiteralExpression(m.initializer.operand)))) {

```

Comment:
I think we should reevaluate the code here. Can we use `resolver.isLiteralConstDeclaration` instead ? That is usually used to print the initializer for constants, but it seems like it should be used here as well (just allow it to take enum members as well)

Reply:
Did you mean like this? Can you take another look?